skimage/measure/_regionprops.py

```diff
@@ -179,6 +195,8 @@ def image(self):
 
     @_cached_property
     def inertia_tensor(self):
+        if self._ndim > 2:
+            raise NotImplementedError('not implemented for 3D images')
```

Comment:
This code is repeated in all functions and should be extracted into a separate method in my opinion.